lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -638,7 +637,7 @@ def setUpClass(cls):
                 metadata={'graded': True, 'format': 'Homework'})
             for _ in xrange(4)
         ]
-        problems = [
+        cls.problems = [
```

Comment:
Are you sure this list is even used anywhere? It couldn't have been accessed before.